### DISCUSSION TITLE
Delete whole tenant?

### DISCUSSION THREAD

**Discussion by jonaz** [grafana/mimir#6051] [Help and support] [answered]
After reading the HTTP API docs i think this will delete ONLY tenant1 and not other tenants. 

```
curl -X POST -H "X-Scope-OrgID: tenant1" http://mimirworker01:8080/compactor/delete_tenant
```

Just wanted to make sure this does not delete our other tenants with 1 year of data in it :)

**Comment by dimitarvdimitrov** [comment, marked as answer]
that's right. Only the tenant in `X-Scope-OrgID` is deleted. If you omit providing the header, then Mimir will respond with HTTP 401 and not take any action
